package.json

```diff
@@ -12,6 +12,9 @@
   "author": "Billy Janitsch <billyjanitsch@gmail.com> (http://billyjanitsch.com)",
   "license": "MIT",
   "repository": "kensho/babel-preset-kensho",
+  "peerDependencies": {
+    "@babel/runtime": "^7.3.1"
```

Comment:
Any reason for this version in particular, as opposed to something like `^7.3.0`?

Reply:
No reason, just used the latest version. But I'll downgrade -- that makes sense.